techdocs/docs/using-the-package/troubleshooting.md

```diff
@@ -0,0 +1,39 @@
+# Troubleshooting
+
+Sometimes issues may arise when using the packages for the first time, or when updating to a new version. Use this guide to help troubleshoot the problem.
+
+## Initial Setup Checklist
+
+1. Ensure you follow the Quick Start Guides closely for each package.
+
+  - [SSO React - Quick Start Guide](../getting-started/quick-start-guide.md)
+  - [SSO Express - Quick Start Guide](https://developer.gov.bc.ca/docs/default/component/citz-imb-sso-express-npm-package/getting-started/quick-start-guide/)
```

Comment:
this link doesn't seem to work

Reply:
It wont work until this is in production.